site/sass/globals/extends/_type.scss

```diff
@@ -22,21 +22,21 @@
 
   &--h2 {
     @extend %type--header;
-    @include flexFont((24px 33.6px), (32px 41.6px), (38px 45.6px));
+    @include flexFont((24px 33.6px), (28px 36.4px), (32px 38.4px));
```

Comment:
I don't mind these are changing, I'm just curious why they are; I thought we just approved the font sizes for these?

Reply:
Design updates came through without my knowledge while our typography PR was pending. I didn't catch them until working through modules that used the individual updated treatments.